platform/ios/src/MGLMapView.mm

```diff
@@ -832,23 +842,46 @@ - (void)updateConstraintsForOrnament:(UIView *)view
                                 size:(CGSize)size
                              margins:(CGPoint)margins {
     NSMutableArray *updatedConstraints = [NSMutableArray array];
+    UIEdgeInsets inset = UIEdgeInsetsZero;
+    
+    BOOL automaticallyAdjustContentInset;
+    if (_automaticallyAdjustContentInsetHolder) {
+        automaticallyAdjustContentInset = _automaticallyAdjustContentInsetHolder.boolValue;
+    } else {
+        UIViewController *viewController = [self rootViewController];
+        automaticallyAdjustContentInset = viewController.automaticallyAdjustsScrollViewInsets;
+    }
+    
+    if (! automaticallyAdjustContentInset) {
```

Comment:
Should this be `if (automaticallyAdjustContentInset)`?

Reply:
If I had an `else` option it would be. In this case I will subtract the safe areas only if `automaticallyAdjustContentInset` is `NO`

Reply:
Thanks for clarifying, I had misread the `-` for a `+` 😄